doc/release/1.10.0-notes.rst

```diff
@@ -109,21 +309,828 @@ Backwards incompatible changes
 *************
 Other changes
 *************
+- `scipy.stats.bootstrap`, `scipy.stats.permutation_test`, and
+  `scipy.stats.monte_carlo_test` now automatically detect whether the provided
+  ``statistic`` is vectorized by looking for an ``axis`` parameter in the
+  signature of ``statistic``. If an ``axis`` parameter is present in
+  ``statistic`` but should not be relied on for vectorized calls, users must
+  pass option ``vectorized==False`` explicitly.
+- `scipy.stats.multivariate_normal` will now raise a ``ValueError`` when the
+  covariance matrix is not positive semidefinite, regardless of which method
+  is called.
 
 
 
 *******
 Authors
 *******
 
+* h-vetinari (10)
```

Comment:
I know it's not new, but maybe mention (here and in the template) what the number in parentheses is. Commits, I'm guessing?